src/components/ConnectionAlert.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { compose, lifecycle } from "recompose";
+import { Alert, Button, Fade } from "react-bootstrap";
+import { i18n } from "../js/i18n";
+
+const enhance = compose(
```

Comment:
I'm okay with this. But is this the new way of writing React? Why not `class Foo extends React.Component`?

Reply:
The [#Why](https://github.com/acdlite/recompose#why) section of `recompose` should explain it clearly.

In my experience, when we could ONLY focus on the props for the component interfaces, it makes it cleaner to express the component and easier for future refactoring.